### DISCUSSION TITLE
Discussione generale strava-tools

### DISCUSSION THREAD

**Discussion by AndreFra96** [AndreFra96/strava-tools#6] [General]
## 👋 Ciaoo!
Utilizziamo questa sezione per discussioni generali che non sono relative ad uno specifico issue.

**Comment by AndreFra96** [comment]
@rafaelventura204 ho creato questa sezione per le discussioni generali, utilizziamola per comunicazioni che non sono relative ad uno specifico issue.

L'ho creata per avvisarti che ho aggiunto delle regole nella repository:
- Non è possibile effettuare push direttamente nel branch main (deve essere fatta una pull request)
- Solo io posso effettuare il merge della pull request all'interno del branch main in seguito alla revisione

Ciaooooo

**Comment by rafaelventura204** [comment]
i nomi  come devono essere dati in camelCase o snake_case? tutto in inglese?

**Reply by AndreFra96** [reply]
per l'inglese direi di si, ci fa bene allenarci un po!

Per quanto riguarda camelCase o snake_case potremmo definire delle regole, a me piacerebbe usare:

- camelCase per i nomi delle variabili e delle funzioni
- SCREAMING_SNAKE_CASE per le variabili d'ambiente
- PascalCase per i nomi dei componenti e delle pagine

sei d'accordo?

**Reply by rafaelventura204** [reply]
ok va bene!

**Comment by rafaelventura204** [comment]
prima di cancellare il branch "pulsante_indietro" devo aspettare che vada a buon fine la pull request? o posso procedere cancellando il branch e creare uno nuovo per la nuova implementazione (#2)?

**Reply by AndreFra96** [reply]
Ti consiglio di aspettare che la pull request vada a buon fine, così se c'è qualche modifica da fare non devi scaricarlo nuovamente.

**Reply by rafaelventura204** [reply]
ho già sbagliato allora... volevo provare a vedere se potevo aprire più branch e ho aperto anche quello per il secondo task

**Reply by AndreFra96** [reply]
nono ma hai fatto bene.

Intendevo solo che ti conviene aspettare a cancellare il primo branch perchè nel caso in cui la review non vada a buon fine (ad esempio c'è qualcosa da modificare) poi devi riscaricarlo ma non è niente di chè.

Puoi avere anche più branch attivi contemporaneamente se stai lavorando su cose diverse, va benissimo.

**Reply by rafaelventura204** [reply]
aah ok capito perfetto!

**Comment by rafaelventura204** [comment]
mi sono informato su un pò di cose, con la tua spiegazione di ieri è stato tutto molto più facile!!

**Reply by AndreFra96** [reply]
Grandeee, mi fa piacere! 💪🏻
